scripts/helpers/index.js

```diff
@@ -19,3 +19,8 @@ export {
 	truncateMiddle,
 	unescapeHTML
 } from './text-helpers';
+export {
+	lockPostEditing,
+	unlockPostEditing,
+	lockIfUndefined
+} from './lock-editing';
```

Comment:
this is specific to the editor only not the frontend so please put this in the editor folder not helpers folder and also export order should be alphabetical order. tnx